nixos/modules/services/networking/nat.nix

```diff
@@ -99,6 +131,18 @@ in
         '';
     };
 
+    networking.nat.internalIPs6 = mkOption {
```

Comment:
`internalIPs6` looks weird. I would call it `internalIPv6s` or`internalIPv6Ranges`.